pmd-apex/src/main/resources/rulesets/apex/apexunit.xml

```diff
@@ -9,6 +9,7 @@
     </description>
 
     <rule ref="category/apex/bestpractices.xml/ApexUnitTestClassShouldHaveAsserts" deprecated="true" />
+    <rule ref="category/apex/bestpractices.xml/ApexUnitTestClassShouldHaveRunAs" deprecated="true" />
```

Comment:
This ruleset is deprecated anyway, we don't need to maintain it, so we can leave this file untouched.

https://pmd.github.io/latest/pmd_rules_apex.html#additional-rulesets

> Deprecated This ruleset is for backwards compatibility.